Makefile

```diff
@@ -179,6 +179,7 @@ COCKPIT_REPO_FILES = \
 	$(NULL)
 
 COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
+# 270
```

Comment:
This was meant to go on the same line as the sha, like

`COCKPIT_REPO_COMMIT = 8f48c4740593e4eb08e1f40ffd98dbeb09f5c14a  # 270`

That's why we have the `$(strip)` below.

Reply:
Done.